yarn.lock

```diff
@@ -4025,11 +4025,6 @@ babel-plugin-polyfill-regenerator@^0.2.2:
   dependencies:
     "@babel/helper-define-polyfill-provider" "^0.2.2"
 
-babel-plugin-remove-graphql-queries@2.7.2:
```

Comment:
Is this removal intentional?

Reply:
definitely not, let me revert from master and see what's going on